flink-runtime/src/main/java/org/apache/flink/runtime/metrics/ReporterSetup.java

```diff
@@ -104,14 +105,9 @@ public static ReporterSetup forReporter(String reporterName, MetricConfig metric
 		return createReporterSetup(reporterName, metricConfig, reporter);
 	}
 
-	private static ReporterSetup createReporterSetup(String reporterName, MetricConfig metricConfig, MetricReporter reporter) {
-		LOG.info("Configuring {} with {}.", reporterName, metricConfig);
-		reporter.open(metricConfig);
-
-		return new ReporterSetup(reporterName, metricConfig, reporter);
-	}
-
-	public static List<ReporterSetup> fromConfiguration(final Configuration configuration) {
+	//TODO: add forwarding method fromConfiguration(final Configuration configuration) which passes null as pluginManager?
+	//TODO: add missing javadoc, mention that pluginManager is optional
+	public static List<ReporterSetup> fromConfiguration(final Configuration configuration, final PluginManager pluginManager) {
```

Comment:
This is the method where it will be used. Current PR is only scoped around propagating the PluginManager to this method from all of the endpoints that load metrics system.

Reply:
This is the method where it will be used. Current PR is only scoped around propagating the PluginManager to this method from all of the endpoints that load metrics system.